models/amazon_ads.yml

```diff
@@ -175,8 +175,8 @@ models:
       - name: cost
         description: "{{ doc('cost') }}"
 
-  - name: amazon_ads__portfolio_report
-    description: Each record in this table represents the daily performance at the portfolio level.
+  - name: amazon_ads__portfolio_report 
+    description: Each record in this table represents the daily performance at the portfolio level. 
```

Comment:
It looks like there are a few fields in this table that are not documented. Similarly, there are a few fields documented that are not used in the default version of the package.
![image](https://user-images.githubusercontent.com/74217849/211377208-bfb1381a-65cd-46fa-a46c-e12678475ca1.png)

Reply:
Done.